Week4/app/src/main/java/com/mmoreno/favmovies/model/DataRepository.kt

```diff
@@ -0,0 +1,115 @@
+package com.mmoreno.favmovies.model
```

Comment:
I like how you placed your data into a repository, hopefully this makes the week 5 work easier for you.